pmd-apex/src/test/resources/net/sourceforge/pmd/lang/apex/rule/bestpractices/xml/ApexUnitTestClassShouldHaveAsserts.xml

```diff
@@ -103,7 +103,7 @@ public class Foo {
 @isTest
 public class Foo {
    public static testMethod void testAssertIsTrue() {
-      Assert.isTrue(someCondition);
+      Assert.assertEquals(someCondition);
```

Comment:
I don't understand this change. The test method name is `testAssertIsTrue` but the tested method was changed to `assertEquals` with only one argument. Typo?

Reply:
You're right, I did not update test method name and to be consistent in arguments number, I'll update the PR.